db/migrate/20160224155909_add_registered_at_to_subscription_facet.rb

```diff
@@ -0,0 +1,9 @@
+class AddRegisteredAtToSubscriptionFacet < ActiveRecord::Migration
+  def up
+    add_column(:katello_subscription_facets, :registered_at, :datetime, :null => false, :default => Time.now)
```

Comment:
I suppose what I meant was:  "are the different in a meaningful way".  These values are promptly replaced by the rake script and are only included here to satisfy the not null constraint.